README.md

```diff
@@ -16,7 +16,7 @@ For further developer and user documentation please visit [the wiki](https://git
 ##### Minimum PHP Version
 * PHP >= 5.4
 
-##### Minimum ownCloud Version
+##### Minimum Nextcloud Version
 * >= 8.1
```

Comment:
Is owncloud now officially unsupported by this app? Another way would be to list both Nextcloud and owncloud.